src/modules/profile/components/helpers/RoleField.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
```

Comment:
To fit with the API change this component should prob be `RolesField` and allow for multiple roles to be selected. If you're lazy you can hard code the roles available instead of querying the server, but you should def allow multiple roles. In terms of ui, to represent selected roles you can use the `Chip` component like `ContributorChip` does. (Don't copy it though, its code is much more complicated than this needs to be)